### ISSUE TITLE
[T1] Terminar juego

### ISSUE THREAD

**Issue by Mjtala** [IIC2113-2022-2/Syllabus#12]
Hola! Tengo una duda,
Si me atacan, y tengo una sola carta en mi arsenal la cual puede revertir el ataque. El juego se acaba igual porque la carta pasa a mi pila de descarte y me quedo sin cartas en mi arsenal o no? Eso es lo que entendí con este ejemplo en la tarea. Gracias!
<img width="1132" alt="Screen Shot 2022-10-03 at 21 16 34" src="https://user-images.githubusercontent.com/62020259/193708080-401e09ba-cec9-457a-9722-422c2d8db254.png">

**Comment by Dnpoblete** [comment]
Hola @Mjtala, si es como tu dices.

Mucho éxito! ✨

**Comment by RodrigoToroIcarte** [comment]
Exacto. Si al final de un turno cualquiera de los jugadores no tiene cartas en su arsenal entonces el juego termina (independiente de cómo se llegó a esa situación).
